server/test/unit/server/db/model/test_resources.py

```diff
@@ -1,55 +1,21 @@
 """
 This module contains tests for the pulp.server.db.model.resources module.
 """
-import mock
-import uuid
+import unittest
 
-from ....base import ResourceReservationTests
 from pulp.server.db.model import resources
 
 
-class TestReservedResource(ResourceReservationTests):
+class TestReservedResource(unittest.TestCase):
 
-    def test_unique_indices(self):
-        self.assertEqual(resources.ReservedResource.unique_indices, tuple())
+    def test_attributes(self):
```

Comment:
Please add a test that asserts that `ReservedResource` is an instance of `Document`.